### PR TITLE
8286810: Use public [Double|Float].PRECISION fields in jdk.internal.math.[Double|Float]Consts

### PR DESCRIPTION
**Pull request by rgiulietti**
Please review these simple changes in jdk.internal.math.[Double|Float]Consts

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (1 review required, with at least 1 reviewer)

### Issue
 * [JDK-8286810](https://bugs.openjdk.java.net/browse/JDK-8286810): Use public [Double|Float].PRECISION fields in jdk.internal.math.[Double|Float]Consts

### Reviewers
 * [Brian Burkhalter](https://openjdk.java.net/census#bpb) (@bplb - **Reviewer**) ⚠️ Review applies to [3c6be86c](https://git.openjdk.java.net/jdk/pull/8729/files/3c6be86c7d2f08580eb1cdfdbe83578d3f1cc3ae)
 * [Roger Riggs](https://openjdk.java.net/census#rriggs) (@RogerRiggs - **Reviewer**) ⚠️ Review applies to [3c6be86c](https://git.openjdk.java.net/jdk/pull/8729/files/3c6be86c7d2f08580eb1cdfdbe83578d3f1cc3ae)
 * [Joe Darcy](https://openjdk.java.net/census#darcy) (@jddarcy - **Reviewer**) ⚠️ Review applies to [3c6be86c](https://git.openjdk.java.net/jdk/pull/8729/files/3c6be86c7d2f08580eb1cdfdbe83578d3f1cc3ae)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8729/head:pull/8729` \
`$ git checkout pull/8729`

Update a local copy of the PR: \
`$ git checkout pull/8729` \
`$ git pull https://git.openjdk.java.net/jdk pull/8729/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8729`

View PR using the GUI difftool: \
`$ git pr show -t 8729`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8729.diff">https://git.openjdk.java.net/jdk/pull/8729.diff</a>

</details>

### CODE REVIEW

File: src/java.base/share/classes/jdk/internal/math/DoubleConsts.java [hunk 1]
```diff
@@ -25,6 +25,8 @@
 
 package jdk.internal.math;
 
+import static java.lang.Double.*;
```
[the remark this answers is not in the document]
Comment (bplb) [reply src/java.base/share/classes/jdk/internal/math/DoubleConsts.java:28]: I concur.

**Comment by rgiulietti** [comment]
Adjusted

**Review by bplb — APPROVED** [review]

**Review by RogerRiggs — APPROVED** [review]
LGTM, thanks

**Review by jddarcy — APPROVED** [review]
Looks fine; at your option, add a comment somewhere to setting the bias fields to explicitly indicate what the value is.

**Pull request closed without merging**